clcache.py

```diff
@@ -69,6 +69,7 @@
 
 def printBinary(stream, rawData):
     stream.buffer.write(rawData)
+    stream.flush()
```

Comment:
I see -- it would be great if you could include this in the commit log (i.e. explain why you needed to do it, what would happen if this was *not* done) to simplify future debugging in case someone complains about e.g. `printBinary` being too slow because it flushes unnecessarily ;-)